overlay/validation/scikitlearn/__init__.py

```diff
@@ -166,7 +167,14 @@ def validate_model(
     # evaluate model
     #score = model.score(features_df, label_df)
     profiler.stop()
-    variance_of_residuals = welford_variance_calculator.var_p
+    # variance_of_residuals = welford_variance_calculator.var_p
+    variance_of_residuals = 0
 
     info(f"Evaluation results for GISJOIN {gis_join}: {loss}")
+    info(f"gis_join: {gis_join}")
+    info(f"allocation: {allocation}")
+    info(f"loss: {loss}")
+    info(f"variance_of_residuals: {variance_of_residuals}")
```

Comment:
`variance_of_residuals` is an NumPy array for some reason and it's throwing an error at the parent class.